documentation/modules/ROOT/pages/dev-services.adoc

```diff
@@ -0,0 +1,221 @@
+
+You can check to see that the fruit endpoint is still working, but in this case the data is coming from the postgresql database. 
+
+[.console-input]
+[source,bash,subs="+macros,+attributes"]
+----
+curl localhost:8080/fruit?season=Spring
+----
+
+[.console-output]
+[source,json]
+----
+[
+  {
+    "id": 1,
+    "name": "Mango",
+    "season": "Spring"
+  },
+  {
+    "id": 2,
+    "name": "Strawberry",
+    "season": "Spring"
+  }
+]
+----
+
+
+== [Optional] Re-deploy to Kubernetes
+
+If you would like to redeploy to Kubernetes, this time you will need to provision a Postgresql database because as mentioned before, Dev Services are only enabled in dev/test mode.
+
+We will also need to add credentials to connect to the external database. Make sure you have the following properties set:
+
+[#quarkus-pgsql-config]
+[.console-input]
+[source,config,subs="+macros,+attributes"]
+----
+%prod.quarkus.datasource.password=quarkus
+%prod.quarkus.datasource.username=quarkus
+%prod.quarkus.datasource.db-kind=postgresql
+%prod.quarkus.datasource.jdbc.url=jdbc:postgresql://postgresql/quarkus
+quarkus.hibernate-orm.database.generation=drop-and-create
+quarkus.hibernate-orm.sql-load-script=import.sql
+----
+
+NOTE: We added a %prod. prefix to some of the properties.  This prefix makes it so these property will only be evaluated with the (default) prod profile.  In dev mode these values will be ignored, thus triggering the Dev Services creation.
+
+=== Create a postgresql database
+
+There are several ways to deploy a Postgresql Database to Kubernetes.  If you're using Openshift, you could create one easily through the UI (Developer Perspective > +Add > Database > PostgreSQL).  Just make sure your database name, username and password match up with what you have configured in your application.properties or secrets.  
+
+You can also apply the following Kubernetes manifest to deploy a simple ephemeral instance:
+
+[.console-input]
+[source,bash,subs="+macros,+attributes"]
+----
+cat <<EOF | kubectl apply -f -
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: postgresql
+spec:
+  selector:
+    matchLabels:
+      app: postgresql
+  template:
+    metadata:
+      labels:
+        app: postgresql
+    spec:
+      containers:
+        - name: postgresql
+          image: quay.io/redhatdemo/openshift-pgsql12-primary:centos7
+          imagePullPolicy: Always
+          ports:
+            - name: tcp
+              containerPort: 5432
+          env:
+            - name: PG_USER_PASSWORD
+              value: quarkus
+            - name: PG_USER_NAME
+              value: quarkus
+            - name: PG_DATABASE
+              value: quarkus
+            - name: PG_NETWORK_MASK
+              value: all
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: postgresql
+spec:
+  ports:
+    - name: pgsql
+      protocol: TCP
+      port: 5432
+      targetPort: 5432
+  type: ClusterIP
+  selector:
+    app: postgresql
+EOF
+----
+
```

Comment:
I will add the kubectl apply command explicitly.

Reply:
@lordofthejars  You mean to create the yaml file first and then apply it ?

Reply:
Yes

Reply:
Ok. I made the changes, let me know